Testing/SystemTests/lib/systemtests/stresstesting.py

```diff
@@ -71,6 +72,16 @@ def __init__(self):
         from mantid.kernel import MemoryStats
         self.memory = MemoryStats().residentMem()/1024
 
+    def setSearchSaveDirectories(self, core_id, saveDir):
+        '''
+        Modify the defaultsave.directory to a path specific to each thread
+        '''
+        from mantid.kernel import config
+        config['datasearch.directories'] += "/core-%i" % (core_id)
+        config['defaultsave.directory'] = "%s/core-%i" % (saveDir,core_id)
```

Comment:
Are we going to need to warn developers now that `config['defaultsave.directory']` is now only fixed at runtime so it shouldn't be cached in the test?